hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -5119,7 +5119,14 @@
     for storing directory snapshot diffs. By default, value is set to 10.
   </description>
 </property>
-
+<property>
+  <name>dfs.namenode.snapshot.deletion.ordered</name>
```

Comment:
@arp7 seems to suggest not to add the conf in hdfs-default.xml.

Reply:
Reverted this change in the updated patch